src/components/ProductCardList/ProductCardList.spec.js

```diff
@@ -0,0 +1,113 @@
+import { render, screen } from '@testing-library/react'
+import ProductCardList from './ProductCardList'
+
+const exampleData = [
```

Comment:
You don't render all this information in your cards yet. Maybe just have the `name` and `description`? Then this is not so big!